go/vt/sidecardb/schema/onlineddl/schema_migrations.sql

```diff
@@ -42,6 +42,7 @@ CREATE TABLE IF NOT EXISTS schema_migrations
     `migration_context`               varchar(1024)    NOT NULL DEFAULT '',
     `ddl_action`                      varchar(16)      NOT NULL DEFAULT '',
     `message`                         text             NOT NULL,
+    `message_timestamp`               timestamp(6)     NULL     DEFAULT NULL,
```

Comment:
why timestamp(6) and not a normal timestamp like other columns?

Reply:
Practice tells me that I always end up "upgrading" a `timetstamp` to `timestamp(6)`. I just appreciate the granularity, especially when it comes to critical points such as cutover where it can really clarify what's going on.